src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/BenefitCode.java

```diff
@@ -60,7 +60,11 @@ public enum BenefitCode {
 
     public static BenefitCode getBenefitCode(String ccdReference) {
         if (isNotBlank(ccdReference) && ccdReference.trim().matches("\\d+")) {
-            return getBenefitCode(Integer.parseInt(ccdReference.trim()));
+            try {
+                return getBenefitCode(Integer.parseInt(ccdReference.trim()));
+            } catch (NumberFormatException exception) {

```

Comment:
I don't get why we're catching these now..

Reply:
Like Waqas said, why are we catching this?
I guess it is a checked exception and SHOULD never be thrown so would rather not declare it as throws.
So rather than catch(...){ return null;}, given you return null at the end on failure anyway, you'd be better off ignoring the error but you should log it as error given you don't expect it to EVER be thrown.

Reply:
Good point, we could let the exception bubble up or let HMC/LA return an error if something vital is missing

Reply:
I thought I removed this, ill do it now